DESCRIPTION

```diff
@@ -14,18 +14,16 @@ Authors@R: c(person("Sheng", "Luo", role="aut"),
 Maintainer: Jiajie Chen <jiajie.chen@pennmedicine.upenn.edu>
 Description: Multiple 2 by 2 tables often arise in meta-analysis which combines statistical evidence from multiple studies. Two risks within the same study are possibly correlated because they share some common factors such as environment and population structure. This package implements a set of novel Bayesian approaches for multivariate meta analysis when the risks within the same study are independent or correlated. The exact posterior inference of odds ratio, relative risk, and risk difference given either a single 2 by 2 table or multiple 2 by 2 tables is provided. Luo, Chen, Su, Chu, (2014) <doi:10.18637/jss.v056.i11>, Chen, Luo, (2011) <doi:10.1002/sim.4248>, Chen, Chu, Luo, Nie, Chen, (2015) <doi:10.1177/0962280211430889>, Chen, Luo, Chu, Su, Nie, (2014) <doi:10.1080/03610926.2012.700379>, Chen, Luo, Chu, Wei, (2013) <doi:10.1080/19466315.2013.791483>.
 Depends: R (>= 2.10.0)
-Imports: aod,ggplot2
+Imports: aod,ggplot2,stats
```

Comment:
why package stats need to be imported here?

Reply:
Some of the functions from the stats package have been in use. This can be removed, as we have already mentioned it in the namespace. I just have it here as a reminder.  

Do you want to remove it?

Reply:
that's fine.